### PR TITLE
PWA-378 Remove currency from locale. Use shop config to get currency …

### PR DESCRIPTION
**Pull request by alexbridge**
…fo filters

### CODE REVIEW

File: components/FilterBar/components/Content/index.jsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { compareObjects } from '@shopgate/pwa-common/helpers/redux';
+/** @type {{currency: string}} */
```
Comment (devbucket) [inline components/FilterBar/components/Content/index.jsx]: Please remove this line. We don't have this anywhere. No reason to start it now.

File: pages/Filter/components/PriceRangeSlider/index.jsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import isEqual from 'lodash/isEqual';
+/** @type {{currency: string}} */
```
Comment (devbucket) [inline pages/Filter/components/PriceRangeSlider/index.jsx]: Please remove this line. We don't have this anywhere. No reason to start it now.

File: components/FilterBar/components/Content/index.jsx (hunk 1 above)
Comment (alexbridge) [reply components/FilterBar/components/Content/index.jsx]: +

File: pages/Filter/components/PriceRangeSlider/index.jsx (hunk 2 above)
Comment (alexbridge) [reply pages/Filter/components/PriceRangeSlider/index.jsx]: +

**Pull request merged**